xml/security_firewall.xml

```diff
@@ -271,535 +271,400 @@
+    </tip>
+
+   </sect3>
+
+   <sect3>
+    <title>Lockdown Mode</title>
+
+    <para>
+     &firewalld; offers a <emphasis>lockdown mode</emphasis> that prevents
+     changes to the firewall rules while being active. Since applications can
+     automatically change the firewall rules via the D-Bus interface and
+     depending on the PolicyKit rules regular users may be able to do the same
+     it can be helpful to prevent changes in some situations. You can find more
+     information about this in
+     <link xlink:href="https://fedoraproject.org/wiki/Features/FirewalldLockdown"/>.
+    </para>
+
+    <para>
+     It is important to understand, however, that the lockdown mode feature
+     provides no real security but merely protection against accidental or
+     benign attempts to change the firewall. The way the lockdown mode is
+     currently implemented in &firewalld; provides no security against malicious
+     intent as is pointed out in
+     <link xlink:href="http://seclists.org/oss-sec/2017/q3/139"/>.
+    </para>
+
+   </sect3>
+
+   <sect3>
+    <title>Adding Custom <command>iptables</command> Rules</title>
+
+    <para>
+     &firewalld; claims exclusive control over the host's
+     <systemitem>netfilter</systemitem> rules. You should never modify
+     firewall rules using other tools like <command>iptables</command>. Doing
+     so could confuse &firewalld; and break security or functionality.
+    </para>
+
+    <para>
+     If you need to add custom firewall rules that aren't covered by
+     &firewalld; features then there are two ways to do so. To directly pass
+     raw <command>iptables</command> syntax you can use the
+     <literal>--direct</literal> option. It expects the table, chain and
+     priority as initial arguments and the rest of the command line is passed
+     as is to <command>iptables</command>. The following example adds a
+     connection tracking rule for the forwarding filter table:
+    </para>
+
+    <screen>&prompt.root;firewall-cmd  --direct --add-rule ipv4 filter FORWARD 0 -i eth0 -o eth1 \
+    -p tcp --dport 80 -m state --state NEW,RELATED,ESTABLISHED -j ACCEPT</screen>
+
+    <para>
+     Additionally &firewalld; implements so called <emphasis>rich
+      rules</emphasis>, an extended syntax for specifying
+     <command>iptables</command> rules in an easier way. You can find the
+     syntax specification in
+     <link xlink:href="http://www.firewalld.org/documentation/man-pages/firewalld.richlanguage.html"/>.
+     the following example drops all IPv4 packets originating from a certain
+     source address:
+    </para>
+
+    <screen>&prompt.root;firewall-cmd --zone=public --add-rich-rule='rule family="ipv4" \
+    source address="192.168.2.4" drop'</screen>
+   </sect3>
+
+   <sect3>
+    <title>Routing, Forwarding and Masquerading</title>
+
+    <para>
+     &firewalld; was originally designed to act as a desktop firewall and as
```

Comment:
@mgerstner do we need that firewalld is okayish for servers? This sounds like firewalld is only suitable for desktops.

Reply:
Yeah that's a difficult spot. What we want to communicate is that _firewalld_ is not designed to run as a fully fledged router. Maybe we should rephrase to something like "is not designed to implement a production router".